src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -125,7 +124,9 @@ export class AccessibleBufferWidget extends DisposableStore {
 			}
 		}));
 		this.add(this._bufferEditor.onDidFocusEditorText(async () => {
-			if (this._inQuickPick) {
+			if (this._accessibleBuffer.classList.contains(CssClass.Active)) {
```

Comment:
You could pull this into a helper `isFocused` function do the check is only defined once

Reply:
yes, already did that in a different branch i'm working on now 👍🏼